app/models/form/lettings/pages/first_time_property_let_as_social_housing.rb

```diff
@@ -2,7 +2,7 @@ class Form::Lettings::Pages::FirstTimePropertyLetAsSocialHousing < ::Form::Page
   def initialize(id, hsh, subsection)
     super
     @id = "first_time_property_let_as_social_housing"
-    @depends_on = [{ "renewal" => 0 }]
+    @depends_on = [{ "not_renewal?" => true }]
```

Comment:
```suggestion
    @depends_on = [{ "renewal" => 0 }]
```

```suggestion
    @depends_on = [{ "is_renewal?" => false }]
```

Why do you need a method for this?

Reply:
I considered using `is_renewal? => false` but in theory renewal could have values 1, 0, nil and I would like to target 0 rather than 0 or nil.
However, in practice renewal is a required question in the setup section so practically `is_renewal? => false` is probably fine?

Reply:
How about?

```ruby
@depends_on = [{ "renewal" => 0 }]
```

Reply:
I can do that if you feel strongly. I'm on a small crusade to remove all conditions that look like that that I come across as I find it pretty obscure to know what a particular code represents.
In my dreams we would replace all of them only with things that should be true and @depends_on would be a 2d array rather than an array of hashes, but I'll try not to get too over-excited.